config/local_docker/.genesis.env

```diff
@@ -1,18 +1,13 @@
 BASE_GENESIS_PATH=./base_genesis.json
 GENESIS_PATH=./genesis.json
-BASE_ROLLUP_CFG_PATH=./base_sp_local_devnet.json
-ROLLUP_CFG_PATH=./sp_local_devnet.json
-
-SEQUENCER_ADDRESS=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
-VALIDATOR_ADDRESS=0x70997970c51812dc3a010c7d01b50e0d17dc79c8
-DEPLOYER_ADDRESS=0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC
-
-ADDR=0.0.0.0
-L1_ENDPOINT=ws://$ADDR:8545
+BASE_ROLLUP_CFG_PATH=./base_sp_rollup.json
+ROLLUP_CFG_PATH=./sp_rollup.json
 
+L1_NETWORK=localhost
+L1_ENDPOINT=ws://0.0.0.0:8545
 L1_STACK=geth
-L1_DEPLOY=true
```

Comment:
we should leave this for the docker config. or add a call to the deploy script in docker-compose if you prefer that

Reply:
added a `-d` flag instead